guides/v2.2/config-guide/varnish/config-varnish.md

```diff
@@ -13,51 +10,49 @@ functional_areas:
+	*	[Install Varnish]
+	*	[Configure Varnish and your web server]
+	*	[Configure Magento to use Varnish]
+	*	[Configure Magento to use multiple Varnish instances]
+	*	[Final verification]
 *	Use Varnish:
-	*	<a href="{{ page.baseurl }}/config-guide/varnish/use-varnish-cache.html">How Magento cache clearing works with Varnish</a>
-	*	<a href="{{ page.baseurl }}/config-guide/varnish/use-varnish-cache-how.html">How Varnish caching works</a>
-*	[Troubleshooting 503 (Backend Fetch Failed) errors]({{ page.baseurl }}/config-guide/varnish/tshoot-varnish-503.html)
+	*	[How Magento cache clearing works with Varnish]
+	*	[How Varnish caching works]
+*	[Troubleshooting 503 (Backend Fetch Failed) errors]
 
-<h2 id="config-varnish-over">Overview of the Varnish solution</h2>
-<a href="https://www.varnish-cache.org/" target="_blank">Varnish Cache</a> is an open source web application accelerator (also referred to as an *HTTP accelerator* or *caching HTTP reverse proxy*). Varnish stores (or caches) files or fragments of files in memory; this enables Varnish to reduce the response time and network bandwidth consumption on future, equivalent requests. Unlike web servers like Apache and nginx, Varnish was designed for use exclusively with the HTTP protocol.
+## Overview of the Varnish solution {#config-varnish-over}
+[Varnish Cache] is an open source web application accelerator (also referred to as an *HTTP accelerator* or *caching HTTP reverse proxy*). Varnish stores (or caches) files or fragments of files in memory; this enables Varnish to reduce the response time and network bandwidth consumption on future, equivalent requests. Unlike web servers like Apache and nginx, Varnish was designed for use exclusively with the HTTP protocol.
 
 Magento 2 supports Varnish 4.x and 5.0.
 
-<div class="bs-callout bs-callout-warning">
-    <p>We <em>strongly recommend</em> you use Varnish in production. The built-in full-page caching (to either the file system or <a href="{{ page.baseurl }}/config-guide/database/database.html">database</a>) is much slower than Varnish, and Varnish is designed to accelerate HTTP traffic.</p>
-</div>
+{:.bs-callout .bs-callout-warning}
+We _strongly recommend_ you use Varnish in production. The built-in full-page caching (to either the file system or [database]) is much slower than Varnish, and Varnish is designed to accelerate HTTP traffic.
 
 For more information about Varnish, see:
 
-*	<a href="https://en.wikipedia.org/wiki/Varnish_%28software%29" target="_blank">wikipedia</a>
-*	<a href="https://www.varnish-cache.org/docs/trunk/users-guide/intro.html" target="_blank">The Big Varnish Picture</a>
-*	<a href="https://www.varnish-cache.org/docs/trunk/reference/varnishd.html#ref-varnishd-options" target="_blank">Varnish startup options</a>
-*	<a href="https://www.varnish-cache.org/docs/trunk/users-guide/performance.html#users-performance" target="_blank">Varnish and Website Performance</a>
+*	[wikipedia]
```

Comment:
Capitalize "Wikipedia" - in other file as well.

Reply:
Fixed